src/include/stir/ProjData.h

```diff
@@ -151,6 +152,8 @@ class ProjData : public ExamData
   //! Set sinogram
   virtual Succeeded 
     set_sinogram(const Sinogram<float>&) = 0;
+  // //! Get Bin value
```

Comment:
get rid of these comments. file should be unchanged

Reply:
👍